src/main/scala/io/getclump/Utils.scala

```diff
@@ -0,0 +1,13 @@
+package io.getclump
+
+private[getclump] case class RunOnce[T, U](f: T => Clump[U]) extends (T => Clump[U]) {
```

Comment:
Maybe the new `ClumpExecution` class could help to remove this class? I don't feel comfortable with merging a change that introduces a `var`.